src/components/screens/consul/Polls/PollDetail.js

```diff
@@ -122,19 +122,18 @@ export const PollDetail = ({ data, onRefresh, route, navigation }) => {
             placeholder={texts.consul.comment}
             autoCapitalize="none"
             control={control}
+            chat
           />
-          <WrapperVertical>
-            <Button
-              onPress={handleSubmit(onSubmit)}
-              title={
-                loading ? texts.consul.submittingCommentButton : texts.consul.commentAnswerButton
-              }
-              disabled={loading}
-            />
-          </WrapperVertical>
-        </Wrapper>
-      </WrapperWithOrientation>
-    </SafeAreaViewFlex>
+          <TouchableOpacity
+            onPress={handleSubmit(onSubmit)}
+            style={styles.button}
+            disabled={loading}
+          >
+            <Icon.Send color={colors.primary} size={normalize(16)} />
+          </TouchableOpacity>
```

Comment:
while testing i wondered, if expired polls are ok to be commented, or if this should be turned off as well? how does consul handle that on their website? are comments possible forever?